src/main/java/net/minecraftforge/fluids/capability/ItemFluidContainer.java

```diff
@@ -0,0 +1,30 @@
+package net.minecraftforge.fluids.capability;
+
+import net.minecraft.item.Item;
+import net.minecraft.item.ItemStack;
+import net.minecraft.nbt.NBTTagCompound;
+import net.minecraftforge.common.capabilities.ICapabilityProvider;
+import net.minecraftforge.fluids.capability.templates.FluidTankItemStack;
+
+public class ItemFluidContainer extends Item
```

Comment:
It's basically an extension of the documentation imo, and a nice win for capabilities. If you have the old `ItemFluidContainer` you see it's deprecated. The documentation there links to this new one. Oh wow, look how clean this is compared to the old one!